addons/l10n_id/models/account_move.py

```diff
@@ -0,0 +1,152 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import pytz
+
+from odoo import api, fields, models, _
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    # Used to fetch the QR payment status.
+    l10n_id_qris_invoice_details = fields.Json(
+        string="QRIS Transaction Number",
+        help="Transaction Number stored.",
+        readonly=True,
+        export_string_translation=False,
+    )
```

Comment:
Maybe comment about the structure of the JSON.
Took me a few to understand that it was a list and not a mapping